src/measure_theory/measure_space.lean

```diff
@@ -461,17 +461,8 @@ lt_iff_le_not_le.trans $ and_congr iff.rfl $ by simp only [le_iff, not_forall, n
 theorem lt_iff' : μ < ν ↔ μ ≤ ν ∧ ∃ s, μ s < ν s :=
 lt_iff_le_not_le.trans $ and_congr iff.rfl $ by simp only [le_iff', not_forall, not_le]
 
--- TODO: add typeclasses for `∀ c, monotone ((*) c)` and `∀ c, monotone ((+) c)`
-
-protected lemma add_le_add_left (ν : measure α) (hμ : μ₁ ≤ μ₂) : ν + μ₁ ≤ ν + μ₂ :=
-λ s hs, add_le_add_left (hμ s hs) _
-
-protected lemma add_le_add_right (hμ : μ₁ ≤ μ₂) (ν : measure α) : μ₁ + ν ≤ μ₂ + ν :=
-λ s hs, add_le_add_right (hμ s hs) _
-
-protected lemma add_le_add (hμ : μ₁ ≤ μ₂) (hν : ν₁ ≤ ν₂) :
-  μ₁ + ν₁ ≤ μ₂ + ν₂ :=
-λ s hs, add_le_add (hμ s hs) (hν s hs)
+instance covariant_add_le : covariant_class (measure α) (measure α) (+) (≤) :=
```

Comment:
Don't we also need
```lean
instance covariant_add_le : covariant_class (measure α) (measure α) (flip (+)) (≤) :=
```
Or is that automatic?

Reply:
That is automatic, see [this instance](https://leanprover-community.github.io/mathlib_docs/algebra/covariant_and_contravariant.html#covariant_swap_add_le_of_covariant_add_le). BTW, we use `function.swap` instead of `flip` in the instances; I don't know the reason.